### PR TITLE
add child environments to integration tests

### LINKED ISSUE #11: Tests for environment token
Right now all of our tests sit in the base environment. We need tests to make sure the environment token retrieval is working.

### PR DESCRIPTION
**Pull request by mcg1969**
Fixes #11.

[NOTE: Builds on #22]

Adds child environment tests into test_config.py

- Makes sure tokens were created for each environment
- Verify the file contents match the user-agent contents
- Verify that the tokens are different for each environment

### CODE REVIEW

File: tests/integration/test_config.py [hunk 1]
```diff
@@ -55,34 +78,45 @@ def _config(value):
...
+        cmd = [
+            "conda",
+            "install",
+            "-vvv",
+            "--override-channels",
+            "-c",
+            "https://repo.anaconda.com/pkgs/fakechannel",
+            "fakepackage",
+            "fakepackage",
+        ]
```
Comment (jezdez) [inline tests/integration/test_config.py:90]: That reminds me, integration tests can without issue use actual packages IMO, is it reasonable to use one from repo.anaconda.com, or even run a small test repo server during the tests?

**Review by jezdez — APPROVED** [review]
Nice, I'm gonna call on you from now on to write stream of thought integration tests :)

File: tests/integration/test_config.py (hunk 1 above)
Comment (mcg1969) [reply tests/integration/test_config.py:90]: The reason I'm using a fake channel and a fake package here is to ensure that the index cache doesn't get populated. This ensures that requests to the server are made every single time

Comment (mcg1969) [reply tests/integration/test_config.py:90]: Adding a conda clean --index here would work too I suppose!

**Pull request merged**